src/Go/boardAnalysis/goAI.ts

```diff
@@ -397,9 +397,10 @@ async function getIlluminatiPriorityMove(moves: MoveOptions, rng: number): Promi
 /**
  * Get a move that places a piece to influence (and later control) a corner
  */
-function getCornerMove(board: Board) {
+async function getCornerMove(board: Board) {
   const boardEdge = board[0].length - 1;
   const cornerMax = boardEdge - 2;
+  await sleep(10);
```

Comment:
These are functional changes, not lint changes. I don't think they belong in this PR?
(I'm also not sure I agree with them in general; I don't think sprinkling sleeps through the code like this is a good idea.)

Reply:
I removed these short sleeps and instead just overrode the lint complaint that these methods were async without strictly needing to be.

removing the async from the shorter move lookup methods would require some very big type refactoring in the way I do lazy move type evaluation and caching for performance.